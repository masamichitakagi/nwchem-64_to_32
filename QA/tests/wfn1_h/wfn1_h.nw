echo
start wfn1_h_dat

geometry
  h 0 0 0
end
basis
  * library 6-31g*
end

set wfn1:print_error T
#set wfn1:ncorea 1
set wfn1:nopen  1
set wfn1:maxit  50
set wfn1:t_bath 0.0

task wfn1 energy

set wfn1:print_error T
set wfn1:nopen  1
set wfn1:maxit  1000
set wfn1:t_bath 0.5

task wfn1 energy
