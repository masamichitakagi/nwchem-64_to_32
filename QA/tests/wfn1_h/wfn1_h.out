 argument  1 = /Users/vandam/nwchem-wfn1-dev/QA/../QA/tests/wfn1_h/wfn1_h.nw



============================== echo of input deck ==============================
echo
start wfn1_h_dat

geometry
  h 0 0 0
end
basis
  * library 6-31g*
end

set wfn1:print_error T
#set wfn1:ncorea 1
set wfn1:nopen  1
set wfn1:maxit  50
set wfn1:t_bath 0.0

task wfn1 energy

set wfn1:print_error T
set wfn1:nopen  1
set wfn1:maxit  1000
set wfn1:t_bath 0.5

task wfn1 energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.6
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = 10-4-27-16.user.vpn.bnl.local
    program         = /Users/vandam/nwchem-wfn1-dev/QA/../bin/MACX64/nwchem
    date            = Fri May 26 09:27:54 2017

    compiled        = Fri_May_26_09:27:12_2017
    source          = /Users/vandam/nwchem-wfn1-dev
    nwchem branch   = Development
    nwchem revision = 28658
    ga revision     = 10724
    input           = /Users/vandam/nwchem-wfn1-dev/QA/../QA/tests/wfn1_h/wfn1_h.nw
    prefix          = wfn1_h_dat.
    data base       = /Users/vandam/nwchem-wfn1-dev/QA/scratchdir/wfn1_h_dat.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = /Users/vandam/nwchem-wfn1-dev/QA/scratchdir
  0 scratch   = /Users/vandam/nwchem-wfn1-dev/QA/scratchdir




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 h                    1.0000     0.00000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      h                  1.007825


 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     1
 geometry
 h                     0.00000000     0.00000000     0.00000000

  library name resolved from: environment
  library file name is: </Users/vandam/nwchem-wfn1-dev/QA/../src/basis/libraries/>
  


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           6-31g*                   on all atoms 



                                NWChem WFN1 Module
                                ------------------


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  h (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757

  2 S  1.61277800E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 h                           6-31g*                  2        2   2s



      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -0.49823254

      Non-variational initial energy
      ------------------------------

 Total energy =      -0.498233
 1-e energy   =      -0.498233
 2-e energy   =       0.000000
 HOMO         =      -0.498233
 LUMO         =       0.460935

 iteration              energy           error          err Na          err Nb          err Ca          err Cb
 ---------              ------           -----          ------          ------          ------          ------
         1         -0.49823275        0.153477        0.000000        0.000000        0.000001        0.153477
         2         -0.49823288        0.002467        0.000000        0.000000        0.000000        0.002467
         3         -0.49823291        0.000009        0.000000        0.000000        0.000000        0.000009

 Reached Convergence

 WFN1                     :
 --------------------------
 Total WFN1 energy =          -0.4982329059
 1-Electron energy =          -0.4982329059
 2-Electron energy =           0.0000000000
 Entropy energy    =           0.0000000000
 Nuclear energy    =           0.0000000000

 conventional Hartree-Fock:
 --------------------------
 Total SCF energy  =          -0.4982329036
 2-Electron energy =           0.0000000023


      Natural orbital occupation numbers

          Alpha     Beta
         ------   ------
     1   1.0000   0.0000
     2   0.0000   0.0000

                     WFN1 Final Alpha Natural Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=1.000000D+00  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 8.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -0.665449  1 H  s                  1     -0.427430  1 H  s          

 Vector    2  Occ=5.066114D-09  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -1.257798  1 H  s                  2      1.149752  1 H  s          


                     WFN1 Final Beta Natural Orbital Analysis
                     ----------------------------------------

 Vector    1  Occ=0.000000D+00  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 8.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -0.665449  1 H  s                  1     -0.427430  1 H  s          

 Vector    2  Occ=0.000000D+00  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -1.257798  1 H  s                  2      1.149752  1 H  s          


 Task  times  cpu:        0.2s     wall:        0.2s


                                NWChem Input Module
                                -------------------



                                NWChem WFN1 Module
                                ------------------



      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -0.49823254

      Non-variational initial energy
      ------------------------------

 Total energy =      -0.498233
 1-e energy   =      -0.498233
 2-e energy   =       0.000000
 HOMO         =      -0.498233
 LUMO         =       0.460935

 iteration              energy           error          err Na          err Nb          err Ca          err Cb
 ---------              ------           -----          ------          ------          ------          ------
         1         -0.49978907        0.172149        0.000000        0.000000        0.018669        0.153479
         2         -0.55191475        0.217230        0.000000        0.000000        0.214760        0.002470
         3         -0.79678647        0.021150        0.000000        0.000000        0.021141        0.000009
         4         -0.80753456        0.006751        0.000000        0.000000        0.006746        0.000005
         5         -0.81090737        0.003306        0.000000        0.000000        0.003303        0.000003
         6         -0.81255976        0.001941        0.000000        0.000000        0.001938        0.000002
         7         -0.81353076        0.001268        0.000000        0.000000        0.001266        0.000002
         8         -0.81416577        0.000891        0.000000        0.000000        0.000889        0.000002
         9         -0.81461175        0.000658        0.000000        0.000000        0.000657        0.000001
        10         -0.81494134        0.000505        0.000000        0.000000        0.000504        0.000001
        11         -0.81519442        0.000400        0.000000        0.000000        0.000399        0.000001
        12         -0.81539464        0.000324        0.000000        0.000000        0.000323        0.000001
        13         -0.81555684        0.000268        0.000000        0.000000        0.000267        0.000001
        14         -0.81569084        0.000225        0.000000        0.000000        0.000224        0.000001
        15         -0.81580335        0.000192        0.000000        0.000000        0.000191        0.000001
        16         -0.81589912        0.000165        0.000000        0.000000        0.000164        0.000001
        17         -0.81598161        0.000144        0.000000        0.000000        0.000143        0.000001
        18         -0.81605337        0.000126        0.000000        0.000000        0.000126        0.000001
        19         -0.81611636        0.000112        0.000000        0.000000        0.000111        0.000000
        20         -0.81617210        0.000099        0.000000        0.000000        0.000099        0.000000

 Reached Convergence

 WFN1                     :
 --------------------------
 Total WFN1 energy =          -0.8161720955
 1-Electron energy =          -0.2803418578
 2-Electron energy =           0.0000000000
 Entropy energy    =          -0.5358302378
 Nuclear energy    =           0.0000000000

 conventional Hartree-Fock:
 --------------------------
 Total SCF energy  =          -0.7362460623
 2-Electron energy =           0.0799260333


      Natural orbital occupation numbers

          Alpha     Beta
         ------   ------
     1   0.7728   0.0000
     2   0.2272   0.0000

                     WFN1 Final Alpha Natural Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=7.728332D-01  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 8.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -0.665449  1 H  s                  1     -0.427430  1 H  s          

 Vector    2  Occ=2.271668D-01  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -1.257798  1 H  s                  2      1.149752  1 H  s          


                     WFN1 Final Beta Natural Orbital Analysis
                     ----------------------------------------

 Vector    1  Occ=0.000000D+00  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 8.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -0.665449  1 H  s                  1     -0.427430  1 H  s          

 Vector    2  Occ=0.000000D+00  E= 0.000000D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -1.257798  1 H  s                  2      1.149752  1 H  s          


 Task  times  cpu:        0.1s     wall:        0.1s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:   86       86     9708     1010     3622        0        0        4     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             2.58e+05 3.07e+04 1.15e+05 0.00e+00 0.00e+00 3.20e+01
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 1056 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        16	         5
	current total bytes		         0	         0
	maximum total bytes		     80136	  22508648
	maximum total K-bytes		        81	     22509
	maximum total M-bytes		         1	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
       S. A. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, V. Konjkov, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson,
      R. J. Harrison, J. R. Rehr, M. Dupuis, D. Silverstein, D. M. A. Smith,
            J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.4s     wall:        0.4s
